tensorboard/uploader/uploader_main.py

```diff
@@ -530,12 +530,16 @@ def execute(self, server_info, channel):
                 ("Id", experiment.experiment_id),
                 ("Created", util.format_time(experiment.create_time)),
                 ("Updated", util.format_time(experiment.update_time)),
-                ("Scalars", str(experiment.num_scalars)),
                 ("Runs", str(experiment.num_runs)),
                 ("Tags", str(experiment.num_tags)),
+                ("Scalars", str(experiment.num_scalars)),
             ]
+            if experiment.total_blob_bytes:
```

Comment:
Why is this conditional? Shouldn’t we show that experiments with no
blobs have 0 blob bytes?

If this is intended to account for old servers, it should be handled
with a fieldmask in the response instead, so that “zero” and “unset”
aren’t conflated. But it doesn’t seem terribly important to me that we
treat old servers specially.

Reply:
As I wrote in the PR description:

> Blob bytes is shown only if it's non-zero
> Rationale: The text output is already pretty verbose, requiring tedious manual scrolling to find an experiment. We want to avoid further increasing the size of the text unnecessarily.

Reply:
I thought about it more. I agree it's better to make it unconditional, because the "name" and "description" fields show something even when there is no name or description. It's better to be consistent with that. Done making the change and updating screenshot.